hx_lti_initializer/views.py

```diff
@@ -515,6 +520,16 @@ def annotation_database_create(request):
         headers=headers
     )
 
+    if request.session['is_graded'] and response.status_code == 200:
```

Comment:
This will trigger a key error if **is_graded** wasn't added to the session, so this should probably check to see if the key exists before checking if it's `True`.